### DISCUSSION TITLE
Waypoint to coordinates

### DISCUSSION THREAD

**Discussion by Le-Juan** [geokrety/geokrety-website#649] [Q&A] [answered]
Hi there,

Let me first quickly introduce myself, I'm Jan-Willem, I'm 21 years old and I'm studying computer science.
Currently I'm working on a personal project, pretty similar to geokrety. 
However it is pure for personal use and the main goal is to learn programming in PHP, JavaScript etc. 
I can program a bit, so currently I have simple logging working. However you guys have on your site a really cool feature, where you can fill in the geocache code and it return the coordinates/location of that cache as drop point.

Could you guys give me some insight on how you got that working? 
I would love to add a somewhat similar feature of that to my own project.

I hope you guys can help and I love to hear from you!

Greetings from the Netherlands!
~Jan-Willem

**Comment by kumy** [comment, marked as answer]
Hi @Le-Juan as explained  by @lineflyer in https://github.com/cgeo/cgeo/discussions/10549#discussioncomment-713669

> [we] built an own database based on users inputs, i.e. I can enter a geocode and in case its not found in their DB Geokrety asks to provide the coordinates for this log

**Reply by Le-Juan** [reply]
Ahhh like that... Thanks for the explanation, I'll start making my own database then 😅
